src/command.rs

```diff
@@ -196,6 +199,7 @@ impl Command {
             }
             "mprev" | "month-prev" => return Ok(Command::MonthPrev),
             "mnext" | "month-next" => return Ok(Command::MonthNext),
+            "wq" | "writeandquit" => return Ok(Command::WriteAndQuit),
```

Comment:
i think we can remove the `writeandquit` alias, after all, vim's `wq` is a composition of two commands.